### PR TITLE
[#5] Develop IPMI Library

### PR DESCRIPTION
**Pull request by NassimBtk**
* Developed IPMI core library
* Developed IPMI client
* Updated license headers
* Updated the README.md file
* Updated the index.md in site/markdown

### CODE REVIEW

File: src/site/site.xml [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
 		<groupId>org.sentrysoftware.maven</groupId>
 		<artifactId>sentry-maven-skin</artifactId>
-		<version>6.1.00</version>
+		<version>6.2.00</version>
 	</skin>
 
 	<custom>
 		<noDefaultLinks>true</noDefaultLinks>
-		<keywords>open source,oss</keywords>
+		<keywords>open source,oss, ipmi</keywords>
```
Comment (bertysentry) [inline src/site/site.xml:12]: Remove `open source, oss` and add `verax` and `vxipmi`

File: README.md [hunk 2]
```diff
@@ -31,9 +30,61 @@ But it is strongly recommended to only use [GitHub Actions "Release to Maven Cen
...
 
-License is Apache-2. Each source file must include the Apache-2 header (build will fail otherwise).
+License is GNU General Lesser Public License (LGPL) version 3.0. Each source file includes the LGPL-3 header (build will fail otherwise).
 To update source files with the proper header, simply execute the below command:
 
 ```bash
 mvn license:update-file-header
 ```
+
+## Run IPMI Client inside Java
```
Comment (bertysentry) [inline README.md:40]: Remove this section (it's already in the project documentation), and add a big link at the top of README.md to the site documentation and Javadoc.

File: README.md [hunk 3]
```diff
@@ -1,11 +1,10 @@
-# oss-maven-template
-
+# IPMI Java Client
 ![GitHub release (with filter)](https://img.shields.io/github/v/release/sentrysoftware/oss-maven-template)
 ![Build](https://img.shields.io/github/actions/workflow/status/sentrysoftware/oss-maven-template/deploy.yml)
 ![GitHub top language](https://img.shields.io/github/languages/top/sentrysoftware/oss-maven-template)
 ![License](https://img.shields.io/github/license/sentrysoftware/oss-maven-template)
 
-Repository template for all Sentry open-source Java projects, published on Maven Central.
+The IPMI Java Client is a library that communicates with the IPMI host, fetches FRUs and Sensors information then reports these information in a text output.
```
Comment (bertysentry) [inline README.md:7]: Add:

```md
This project is a fork of the excellent [IPMI Library for Java by Verax Systems](https://veraxsystems.com/ipmi-library-for-java/) ([see also](https://en.wikipedia.org/wiki/Verax_IPMI)). It is however not related to [another fork by rbuckland](https://github.com/rbuckland/ipmilib).

See **[Project Documentation](https://sentrysoftware.github.io/ipmi)** and the [Javadoc]((https://sentrysoftware.github.io/ipmi/javadocs) for more information on how to use this library in your code.
```

File: pom.xml [hunk 4]
```diff
@@ -29,38 +29,60 @@
...
 	</developers>
 
 	<properties>
-		<!-- Java 11 -->
-		<maven.compiler.source>11</maven.compiler.source>
-		<maven.compiler.target>11</maven.compiler.target>
-		<maven.compiler.release>11</maven.compiler.release>
+		<!-- Java 8 -->
+		<java.version>8</java.version>
+		<maven.compiler.source>${java.version}</maven.compiler.source>
```
Comment (bertysentry) [inline pom.xml:83]: Do we still need `<maven.compiler.source>` and `<maven.compiler.target>`?

File: src/site/resources/images/sentry-logo-179x75px.png
Comment (bertysentry) [inline src/site/resources/images/sentry-logo-179x75px.png:1]: GitHub says the image is invalid, weird...

File: src/site/site.xml
Comment (bertysentry) [inline src/site/site.xml:1]: Please add a direct link in the menu to the Javadoc

**Review by bertysentry — CHANGES_REQUESTED** [review]
That's great! A few minor changes required, thank you!

File: pom.xml (hunk 4 above)
Comment (NassimBtk) [reply pom.xml:83]: Good catch, we only need `<maven.compiler.release>8</maven.compiler.release>`

File: src/site/resources/images/sentry-logo-179x75px.png
Comment (NassimBtk) [reply src/site/resources/images/sentry-logo-179x75px.png:1]: Yes, strange, we will add the banner left icon.

File: README.md (hunk 3 above)
Comment (NassimBtk) [reply README.md]: Thanks @bertysentry 😉.

**Review by bertysentry — APPROVED** [review]
Let's go!

**Pull request merged**